docs/reference/query-languages/esql/_snippets/commands/layout/sample.md

```diff
@@ -1,7 +1,7 @@
 ## `SAMPLE` [esql-sample]
 
 ```yaml {applies_to}
-stack: preview 9.1
+stack: preview 9.1.0
```

Comment:
You can use the minor or the full patch as see fit, I don't know if we've had enough usage yet to arrive on an organic consensus

Reply:
Err, I guess this also needs a `serverless: preview` - otherwise it's declared GA in Serverless, right?